src/Kafka/CreateKafkaTopic/components/StepTopicName.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useCallback, useEffect } from "react";
+import { useTranslation } from "react-i18next";
+import {
+  Form,
+  FormGroup,
+  FormSection,
+  Text,
+  TextContent,
+  TextInput,
+  TextVariants,
+} from "@patternfly/react-core";
+import { useValidateTopic } from "../types";
+import type { NewTopic } from "../types";
+
+export type StepTopicNameProps = {
+  newTopicData: NewTopic;
+  onTopicNameChange: (value: NewTopic) => void;
+  topicNameValidated: "error" | "default";
+  onValidationCheck: (value: "error" | "default") => void;
+  invalidText: string;
+  setInvalidText: (value: string) => void;
+};
+
+export const StepTopicName: React.FC<StepTopicNameProps> = ({
+  newTopicData,
+  onTopicNameChange,
+  topicNameValidated,
+  onValidationCheck,
+  invalidText,
+  setInvalidText,
+}) => {
+  const { t } = useTranslation(["create-topic"]);
+  /*eslint-disable */
```

Comment:
es lint was complaining on this line 'Avoid referencing unbound methods which may cause unintentional scoping of `this` '  @riccardo-forina, any tips to fix this?